charge.go

```diff
@@ -19,6 +19,24 @@ const (
 	ChargeFraudStripeReportFraudulent ChargeFraudStripeReport = "fraudulent"
 )
 
+// ChargeLevel3LineItemsParams is the set of parameters that represent a line item on level III data.
+type ChargeLevel3LineItemsParams struct {
+	DiscountAmount     *int64  `form:"discount_amount"`
+	ProductCode        *string `form:"product_code"`
+	ProductDescription *string `form:"product_description"`
+	Quantity           *int64  `form:"quantity"`
+	TaxAmount          *int64  `form:"tax_amount"`
+	UnitCost           *int64  `form:"unit_cost"`
+}
+
+// ChargeLevel3Params is the set of parameters that can be used for the Level III data.
+type ChargeLevel3Params struct {
+	LineItems          []*ChargeLevel3LineItemsParams `form:"line_items"`
+	MerchantReference  *string                        `form:"merchant_reference"`
+	ShippingAddressZip *string                        `form:"shipping_address_zip"`
+	ShippingAmount     *int64                         `form:"shipping_amount"`
```

Comment:
I was just looking at the implementation and noticed that there also seems to be parameters for `customer_reference` and `shipping_from_zip` in there. Do we need those?

Reply:
@brandur-stripe hmmm potentially though those are not documented even in our internal documentation so I think we likely should punt and add those later once the feature is out of beta or asked in one of the languages. What do you think?

Reply:
Yeah, sounds good to me. Thanks.